examples/using-mobx/gatsby-config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Would you mind cleaning up this formatting?

`yarn format-examples` or `npm run format-examples` in the root of the repo should do the trick!